projects/hslayers/src/components/add-data/vector/add-data-vector.component.ts

```diff
@@ -103,6 +103,8 @@ export class HsAddDataVectorComponent {
         workspace: this.hsCommonEndpointsService.endpoints.filter(
           (ep) => ep.type == 'layman'
         )[0]?.user,
+        queryCapabilities:
+          this.url && !['geojson', 'kml'].some((ext) => this.url.endsWith(ext)),
```

Comment:
GeoJSON can have .json extension as well, so perhaps
```suggestion
          this.url && !['json', 'kml'].some((ext) => this.url.endsWith(ext)),
```
?